extension/js/content_scripts/webmail/gmail/gmail-element-replacer.ts

```diff
@@ -640,7 +640,6 @@ export class GmailElementReplacer extends WebmailElementReplacer {
     const legacyDraftReplyRegex = new RegExp(/\[(flowcrypt|cryptup):link:draft_reply:([0-9a-fr\-]+)]/);
     const newReplyBoxes = $('div.nr.tMHS5d, td.amr > div.nr, div.gA td.I5').not('.reply_message_evaluated').filter(':visible').get();
     if (newReplyBoxes.length) {
-      this.replyOption = undefined;
```

Comment:
removing this line caused unexpected draft creation bug reappear - https://github.com/FlowCrypt/flowcrypt-browser/issues/5616#issuecomment-1972897692

Reply:
Ohhh, Yep
I now remember, Thank you for that notice.

Reply:
let's add comment about this issue above this line, so we won't miss it in the future.